judgments/models.py

```diff
@@ -65,9 +37,53 @@ def create_from_node(node):
         matches = SearchMatch.create_from_string(
             etree.tostring(node, encoding="UTF-8").decode("UTF-8")
         )
-        judgment_xml = api_client.get_judgment_xml(uri)
-        judgment = Judgment.create_from_string(judgment_xml)
-
+        namespaces = {
+            "search": "http://marklogic.com/appservices/search",
+            "uk": "https://caselaw.nationalarchives.gov.uk/akn",
+            "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
+        }
+        neutral_citation = (
+            node.xpath("search:extracted/uk:cite", namespaces=namespaces)[0].text
+            if node.xpath("search:extracted/uk:cite", namespaces=namespaces)
+            else ""
+        )
+        court = (
+            node.xpath("search:extracted/uk:court", namespaces=namespaces)[0].text
+            if node.xpath("search:extracted/uk:court", namespaces=namespaces)
+            else ""
+        )
+        metadata_name = (
+            node.xpath("search:extracted/akn:FRBRname/@value", namespaces=namespaces)[0]
+            if node.xpath("search:extracted/akn:FRBRname/@value", namespaces=namespaces)
+            else ""
+        )
+        date = (
+            node.xpath(
+                "search:extracted/akn:FRBRdate[not(@name='transform')]/@date",
```

Comment:
Feels odd to use "not transform date"; there feels like there's potentially an issue if more dates come along (e.g. enrichment). Is this "judgment" vs "decision", and would `(@name='judgment' or @name='decision')` be better?

Reply:
Yes this felt weird to me too. Unfortunately the dates both come out as `search:extracted/akn:FRBRdate`. Adding (@name='judgment' or @name='decision') would work though!

Reply:
Amended